src/platyPS/platyPS.psd1

```diff
@@ -51,7 +51,7 @@ Description = 'Generate PowerShell External Help files from Markdown'
 # RequiredModules = @()
 
 # Assemblies that must be loaded prior to importing this module
-RequiredAssemblies = @('Markdown.MAML.dll','YamlDotNet.dll')
+RequiredAssemblies = @('Markdown.MAML.dll','YamlDotNet.NetStandard.dll')
```

Comment:
Same comment as above. Needs to run on PS 5.1 unless we are explicitly dropping support for it.

Reply:
AFAIK NetStandard is 5.1 compatible.